resources/examples/additional-parameters.html

```diff
@@ -1 +1 @@
-<link href="https://cdn.curlwind.com?classes=p-*:sm|md,m-*:hover">
+<link rel="stylesheet" href="https://cdn.curlwind.com?classes=p-*:sm|md,m-*:hover">
```

Comment:
I don't want to include this attribute as it obfuscates the examples

They should be as plain as possible. Anyone pasting the link in can just add them afterwards, as it'll work normally with or without `rel="stylesheet"`

Reply:
Nevermind, done! 👍